scripts/deploy.sh

```diff
@@ -21,47 +23,61 @@ case "${env}" in
     ;;
 esac
 
-FILE_PATHS=(
+LEGACY_FILE_PATHS=(
```

Comment:
What about keeping a single list of files created according to the suffix? This way you wouldn't have to duplicate function calls and pass global variable names as argument.

```bash
FILE_PATHS=(
  "packages/logs/bundle/datadog-logs.js"
  "packages/rum/bundle/datadog-rum.js"
  "packages/rum-recorder/bundle/datadog-rum-recorder.js"
)

# legacy files do not support suffix
if [[ -z $suffix ]]; then
  FILE_PATHS+=(
    "packages/logs/bundle/datadog-logs-eu.js"
    "packages/logs/bundle/datadog-logs-us.js"
    "packages/rum/bundle/datadog-rum-eu.js"
    "packages/rum/bundle/datadog-rum-us.js"
  )
fi
```

Reply:
Yes, it can be a bit easier